### DISCUSSION TITLE
Application error: a client-side exception has occurred (see the browser console for more information).

### DISCUSSION THREAD

**Discussion by Hongbusi** [vercel/next.js#52671] [Help]
### Summary

When I visit [next official documents](https://nextjs.org/docs), this error often occurs when the route jumps:

![image](https://github.com/vercel/next.js/assets/53554371/050da46f-b0ea-47b9-92a3-6ac8a6ccd021)

The console error content is: 

![image](https://github.com/vercel/next.js/assets/53554371/ae5268c2-8493-4b5a-86ae-639d2023015e)

I have seen some similar discussions, but I didn't find a discussion about accessing official documents and reporting errors. This confuses me. I hope to get some help. Thanks.

**Comment by joaopedrodcf** [comment]
Did you check if you are using google translate to translate the website ? Google translate made that problem on my side and the solution was to disable it in the html tag: translate="no" 

Hopefully this is your problem 🙏 

More information in here: https://github.com/facebook/react/issues/13278#issuecomment-505491484
